src/core/grpc_service.proto

```diff
@@ -741,6 +741,23 @@ message ModelInferResponse
   repeated InferOutputTensor outputs = 5;
 }
 
+//@@
+//@@.. cpp:var:: message ControlResponse
+//@@
+//@@   The control signal for a stream inference response
+//@@
+message ControlResponse
```

Comment:
Does the client library need to know the final response? I don't think so. If we need to add something like this we would need to add it for HTTP as well... I'd rather not have it at all.

Reply:
Python library doesn't the end signal but I believe we might need something like this for C++ client library to handle statistics. I have removed it from this PR.